script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -540,7 +540,7 @@ Akiha isn't childish enough to give chase after her older brother announces he's
 [sha:8f78ce07403bedb91d17909c991fb4c508c98225]{
 -- Page 38, Offset 16665.
 -- 「けど、怒る理由がルールについてなのが秋葉らしいな」
-"But it really is typical of Akiha to be upset about the rules of all things."
+"Being upset about the rules of all things is just like Akiha."
```

Comment:
I think here he meant it's something Akiha would do, not something Akiha used to do.